src/components/Button/index.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { InlineIcon } from '@iconify/react';
 import { Buttons } from './styles';
 
-const Button = ({ children, buttonType, onClick, icon, iconSmall }) => (
+const Button = ({ children, buttonType, onClick, icon }) => (
   <Buttons buttonType={buttonType} onClick={onClick}>
     {children}
     {icon && <InlineIcon icon={icon} />}
```

Comment:
Don't see necessary the Icon on this Button, I think we can Create another with this one To use with icons , the Children Should be Render the Icon

Reply:
Anyway the icon prop is needed because the button doesn't always have the same icon and I think the use of two button components would be redundant.

The other solution that I foresee is to definitely use the button without any icon at all and when the button has an icon just wrap the icon inside of the button.

Is this last thing what you meant? I am very confused here.

Reply:
Yes the Icon should exist in the children if its needed